### PR TITLE
Position setting: use 9slice widget and moved some options

### PR DESCRIPTION
**Pull request by mrbuds**

### CODE REVIEW

File: WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasAnchorButtons.lua [hunk 1]
```diff
@@ -5,13 +5,15 @@ local AceGUI = LibStub and LibStub("AceGUI-3.0", true)
 if not AceGUI or (AceGUI:GetWidgetVersion(Type) or 0) >= Version then return end
```
Comment (InfusOnWoW) [inline WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasAnchorButtons.lua:5]: There's a bunch of mixed indentation. Some lines are indented by tabs, some by spaces

File: WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasAnchorButtons.lua [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      direction
+    )
+
+    local buttonTex = button:CreateTexture()
+    buttonTex:SetAllPoints()
+    buttonTex:SetTexture("Interface\\AddOns\\WeakAuras\\Media\\Textures\\Square_FullWhite.tga")
+    buttonTex:SetVertexColor(0.3, 0.3, 0.3, 1)
+    button:SetNormalTexture(buttonTex)
+    button.tex = buttonTex
+    button.num = index
```
Comment (InfusOnWoW) [inline WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasAnchorButtons.lua:115]: This appears to be some left-over code

File: WeakAurasOptions/WeakAurasOptions.toc [hunk 3]
```diff
@@ -90,8 +90,8 @@ AceGUI-Widgets\AceGUIWidget-WeakAurasSortedDropDown.lua
 AceGUI-Widgets\AceGUIWidget-WeakAurasToolbarButton.lua
 AceGUI-Widgets\AceGUIWidget-WeakAurasTwoColumnDropDown.lua
 AceGUI-Widgets\AceGUIWidget-WeakAurasSnippetButton.lua
+AceGUI-Widgets\AceGUIWidget-WeakAurasAnchorButtons.lua
 AceGUI-Widgets\AceGUIContainer-WeakAurasTreeGroup.lua
-AceGUI-Widgets\AceGUIWidget-WeakAurasSnippetButton.lua
```
Comment (InfusOnWoW) [inline WeakAurasOptions/WeakAurasOptions.toc:94]: This appears to be accidental.

Comment (mrbuds) [reply WeakAurasOptions/WeakAurasOptions.toc:94]: The change doesn't belong to this PR, this lua file was added twice in the .toc (2 lines before)

**Comment by mrbuds** [comment]
fixed indenting, removed leftover code line, and rebased in a single commit

**Pull request merged**